### ISSUE TITLE
Cheats not running in game.

### ISSUE THREAD

**Issue by davesahaj** [danielkrupinski/curiumcheat-csgo#2]
successfully compiled and build the cheats dll, made an exe with the curium injector, used in offline csgo but the cheats console just pops up and closes. Pressing insert does nothing afterwards.

**Comment by danielkrupinski** [comment]
1. Open main menu of CSGO
**1. Put injector and cheat in the same folder**
**1. Rename .dll to curium.dll**
**1. Launch injector as an administrator - console closes itself**
1. Go back to CSGO
1. Wait ~5 seconds for cheat to load
1. Cheat is loaded if there is a sign in top left corner
**Remember to use Enigma Protector on cheat .dll file to make sure cheat is undetected by VAC.**

**Comment by davesahaj** [comment]
Followed the instructions, had some warnings during dll compilation.
 encrypted the dll with enigma protector(full ver)
placed the dll and injector exe in same folder
launched the game and then injector with admin privileges.
console pop ups and closes, nothing happens in game. no sign in top left corner
game crashes randomly after 2-3 minutes.
Could you upload the RELEASE files in the rep? so we could check if the problem is in VS configs. or in the code.

**Comment by davesahaj** [comment]
Followed the instructions, had some warnings during dll compilation.
encrypted the dll with enigma protector(full ver)
placed the dll and injector exe in same folder
launched the game and then injector with admin privileges.
console pop ups and closes, nothing happens in game. no sign in top left corner
game crashes randomly after 2-3 minutes.

**Comment by danielkrupinski** [comment]
Download cheat and injector from here and try it:
https://github.com/danielkrupinski/curiumcheat-csgo/releases
https://github.com/danielkrupinski/curiuminjector-csgo/releases

**Comment by davesahaj** [comment]
yes releases are working fine but not compiled cheats.

**Comment by davesahaj** [comment]
the release version of cheat is working...but in cheats compilation I got errors :
 error C4996 'strcpy': This function or variable may be unsafe.
i used _CRT_SECURE_NO_WARNINGS in preprocessor definitions to solve it.
then I got 399 warnings,

 C4244: '=': conversion from 'double' to 'float', possible loss of data // more then 300 times.
 C4145: '==': relational expression as switch expression; possible confusion with '='
 C4808: case '2' is not a valid value for switch condition of type 'bool'
 C4018: '<': signed/unsigned mismatch
 C4309: 'initializing': truncation of constant value
 C4305: 'initializing': truncation from '__int64' to 'long'

anyway compilation was successful but the file is not working in game.
